examples/turn-server/simple/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.1"
+
+pion-turn:
+  freeswitch:
+    container_name: pion-turn
+     image: pion-turn:${VERSION:-latest}
+    build:
+      context: ./
+    tty: true
+    stdin_open: true
+    environment:
+      - VERSION=${PION_TURN_VERSION:-master}
+      - REALM=${PION_TURN_REALM:-localhost}
+      - USERS=${PION_TURN_USERS:-username=password}
+      - PUBLIC_IP=${PION_TURN_PUBLIC_IP:-127.0.0.1}
+      - UDP_PORT=${PION_TURN_UDP_PORT:-3478}
+    #networks:
+    #  - backend
+    network_mode: host
+    ports:
+      # STUN
+      - "3478:3478"
+      # TURN
+      - "49152-65535:49152-65535"
+    cap_add:
+      - NET_ADMIN
+      - NET_RAW
+   
+#networks:
```

Comment:
Can we remove the `networks` sections here if they are not used?

Reply:
Done